File: library/src/main/java/com/getstream/sdk/chat/interfaces/ClientConnectionCallback.java
package com.getstream.sdk.chat.interfaces;

import com.getstream.sdk.chat.enums.ClientErrorCode;
import com.getstream.sdk.chat.rest.User;

public interface ClientConnectionCallback {

    void onSuccess(User user);

    /**
     *
     * @param errMsg Human readable message
     * @param errCode http status code or {@link ClientErrorCode}
     */
    void onError(String errMsg, int errCode);
}

File: app/src/main/java/io/getstream/chat/android/core/poc/library/ClientConnectionCallback.kt
package io.getstream.chat.android.core.poc.library

interface ClientConnectionCallback {
    fun onSuccess(user: User?)
    /**
     *
     * @param errMsg Human readable message
     * @param errCode http status code or [ClientErrorCode]
     */
    fun onError(errMsg: String?, errCode: Int)
}
